docs/ADOPTERS.md

```diff
@@ -1,7 +1,7 @@
 ## Adopters
 * [Advanced Telematics Systems](https://advancedtelematic.com/en/press-releases/ats-is-integrating-the-uptane-security-framework-for-over-the-air-software-updates-to-connected-vehicles.html)
 * [Airbiquity](https://www.airbiquity.com/news/press-releases/airbiquity-unveils-new-otamatictm-release-further-strengthening-over-air-ota-software-and-data-management-offering-automotive)
-* [Automotive Grade Linux](http://docs.automotivelinux.org/docs/architecture/en/dev/reference/security/05-platform-security.html#agl-platform-software-update)
+* [Automotive Grade Linux](https://docs.automotivelinux.org/docs/architecture/en/dev/reference/security/05-platform-security.html#agl-platform-software-update)
```

Comment:
This link seems dead (at least the path).

Reply:
I have updated to new url : http://docs.automotivelinux.org/master/docs/architecture/en/dev/reference/security/README.html